cpp/include/copc-lib/io/reader.hpp

```diff
@@ -69,14 +67,19 @@ class Reader : public BaseIO
 
     // Constructor helper function, initializes the file and hierarchy
     void InitReader();
-    // Reads file VLRs into vlrs_
-    std::map<uint64_t, las::VlrHeader> ReadVlrs();
+    // Reads file VLRs and EVLRs into vlrs_
+    std::map<uint64_t, las::VlrHeader> ReadVlrHeaders();
+    // Fetchs the map key for a query vlr user and record IDs
+    static uint64_t FetchVlr(const std::map<uint64_t, las::VlrHeader> &vlrs, const std::string &user_id,
+                             uint16_t record_id);
     // Finds and loads the COPC vlr
-    las::CopcVlr ReadCopcData();
+    CopcInfo ReadCopcInfoVlr();
```

Comment:
We shouldn't expose these functions if we also provide the CopcInfo object through base_io, it means they could get it from two places. Maybe only FetchVlr should be exposed, and ReadVlrHeaders() should be called in constructor and they can access those offsets through a property or something.